paywall/src/paywall-script/module.js

```diff
@@ -0,0 +1,5 @@
+import { isUnlocked } from '../utils/isUnlocked'
+
+export default {
```

Comment:
I believe if you don't do a default export you'll be able to reference it as 

```javascript
const paywall = require('@unlock-protocol/paywall')
paywall.isUnlocked // ...
```

Without having the default in between

Reply:
but then how do I `export`?

Reply:
You just `export` without the default keyword

Reply:
oh! of course! let me try!

Reply:
worked :)